swift/Sources/PiranhaKit/InputCommand/PiranhaCommand.swift

```diff
@@ -0,0 +1,88 @@
+/**
+ *    Copyright (c) 2019 Uber Technologies, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+import Foundation
+import ArgumentParser
+import SwiftSyntax
+
+public struct PiranhaCommand: ParsableCommand {
+    
+    public init() { }
+    
+    public static var configuration = CommandConfiguration(subcommands: [CleanupStaleFlagsCommand.self],
+                                                           defaultSubcommand: CleanupStaleFlagsCommand.self)
+}
+
+struct CleanupStaleFlagsCommand: ParsableCommand {
+    
+    static var _commandName: String = "cleanup-stale-flags"
+    
+    @Option(name: [.customShort("s"),
+                   .long],
+            help: "Input Source File that needs to run piranha",
```

Comment:
Input source file for Piranha?

Reply:
Yeah, that does seem better to me too. Thanks for suggesting this - will make the changes!

Reply:
Updated!